pages/live/index.js

```diff
@@ -173,13 +205,26 @@ function _extendLastTranscript (text) {
 function _addNewTranscript (text, partyCode, partyName, startTime) {
   const root = document.querySelector('#transcriptRoot')
   // create nodes
-  const transcriptRoot = root.appendChild(createNode('div', {'class': 'flexRowLeft transcript'}))
-  transcriptRoot.appendChild(createNode('div', {'class': 'flexColumn transcriptTime'}, durationToString(startTime)))
-  const nameContentNode = transcriptRoot.appendChild(createNode('div', {'class': 'transcriptNameContent'}))
+
+  const transcriptRoot = root.appendChild(createNode('div', {'class': 'flexColumn transcript'}))
+
+  let transcriptHeader
+  let contentStyle
+
+  if (partyCode == 0) {// prospect and should be RHS
+    transcriptHeader = transcriptRoot.appendChild(createNode('div', {'class': 'flexRow', 'style': 'margin-left: 20%'}))
+    contentStyle = 'margin-left: 20%'
```

Comment:
i would have expected that the easiest way to align something to the rhs would be with something like `class: 'flexRowRight'`, but i'm fine with this if it looks like what you want it to look like

Reply:
didn't want to align it to the right, want it to be floating a bit to the left